### PR TITLE
Add PyTorch Project Convertor——CRAFT

### PR DESCRIPTION
**Pull request by SunAhong1993**
# Create A Good Pull Request

Please check the follow step before merging this pull request

- [x] Python code style verification
- [x] Review all the code diff by yourself
- [x] All models(TensorFLow/Caffe/ONNX/PyTorch) testing passed
- [ ] Details about your pull request, releated issues

If this PR add new model support, please update `model_zoo.md` and add model to out test model zoos(@wjj19950828)
- [x] New Model Supported
- [ ] No New Model Supported
## PR内容
1. 新增支持[CRAFT](https://github.com/clovaai/CRAFT-pytorch)的转换。
2. 对应的新增相关文档。

### CODE REVIEW

File: x2paddle/project_convertor/pytorch/api_mapper/nn.py [hunk 1]
```diff
@@ -386,10 +386,10 @@ def __init__(self,
...
         super().__init__(func_name, pytorch_api_name, args, kwargs, target_name)
 
-    def delete_attrs(self):
-        delete_key(self.kwargs, "inplace")
-        if len(self.args) > 0:
-            self.args.clear()
+    def run(self):
+        if len(self.args) > 0 or len(self.kwargs) > 0:
+            self.func_name = "x2paddle.torch2paddle.ReLU"
+        return super().run()
```
Comment (jiangjiajun) [inline x2paddle/project_convertor/pytorch/api_mapper/nn.py:392]: 这里怎么有这么一个特殊的判断操作

File: x2paddle/project_convertor/pytorch/models/resnet.py [hunk 2]
```diff
@@ -79,7 +79,7 @@ def __init__(self,
         # Both self.conv1 and self.downsample layers downsample the input when stride != 1
         self.conv1 = conv3x3(inplanes, planes, stride)
         self.bn1 = norm_layer(planes)
-        self.relu = nn.ReLU()
+        self.relu = torch2paddle.ReLU(True)
```
Comment (jiangjiajun) [inline x2paddle/project_convertor/pytorch/models/resnet.py:82]: True是干啥用的

Comment (jiangjiajun) [reply x2paddle/project_convertor/pytorch/models/resnet.py:82]: 哦，原来是Inplace操作

**Review by jiangjiajun — CHANGES_REQUESTED** [review]

File: x2paddle/project_convertor/pytorch/api_mapper/nn.py (hunk 1 above)
Comment (SunAhong1993) [reply x2paddle/project_convertor/pytorch/api_mapper/nn.py:392]: 这里是如果出现`inplace`这一参数，则需要使用封装ReLU。

**Pull request merged**